hooks/useWidth.js

```diff
@@ -0,0 +1,21 @@
+import { useTheme } from "@material-ui/core/styles";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+/**
+ * Be careful using this hook. It only works because the number of
+ * breakpoints in theme is static. It will break once you change the number of
+ * breakpoints. See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level
+ *
+ * @returns {Object} breakpoint of the windows
+ */
+export default function useWidth() {
```

Comment:
I also saw this hook example in the MUI documentation to recrate the original `widthWith` hebaviour , I think it's fine to add it, there might be other usecases for it aswell.